### PR TITLE
Ath.homescreen tab nav

### LINKED ISSUE #2: [FEATURE] add simple Homepage
Thecyberhub-app is a react-native app

We have to build the same home page

Here is our home screen:
photo_2022-10-04_00-06-58

and we want to this exactly same on our home screen
image

website link https://www.thecyberhub.org/
website source code is also available on the GitHub

If you need any help or having doubts, let us know:

Thanks

### LINKED ISSUE #6: Style System
### Detailed description

The following PR proposes the implementation of a simple style system for the app.

### Context

Future contributors can reference this system all over the app for uniformity in styles.

### Possible implementation

#5 already introduces the style system.

### Additional information

Happy to discuss changes to colors etc.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ATH42**
Closes #2
Closes #6 

## Changes
this merge introduce
- simple Home Screen with implemented video playing in background.
- style system

## Note to reviewers
The Video playing in the background has to be implemented via uri,
I used my proprietary upload for testing, this needs to be uploaded again by maintainers for future use.

There is also a poster image needed, this should be shown as long as the video loads in the background.

## Code of Conduct
- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes)
- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by ATH42** [comment]
hey @kabir0x23 , this is ready to be reviewed.
let me know if you need some more help on this project.
regards A

File: screens/HomeScreen.js [hunk 1]
```diff
@@ -1,12 +1,67 @@
...
-                onPress={() => alert('This is the "Home" screen.')}
-                style={{ fontSize: 26, fontWeight: 'bold' }}>Home Screen</Text>
-        </View>
-    );
-}
\ No newline at end of file
+  return (
+    <>
+      <Video
+        style={{ flex: 1, position: 'fixed' }}
```
Comment (kabir0x23) [inline screens/HomeScreen.js:12]:
```css
 position: 'fixed'
```
is creating a problem, like sometimes crashing the expo app
image

Comment (ATH42) [reply screens/HomeScreen.js]: done.

**Review by kabir0x23 — APPROVED** [review]
Awesome.

Thanks for contributing to this open-source project.

**Pull request merged**